src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -30,31 +33,44 @@ const messages = Messages.loadMessages(
     'device-create'
 );
 
-export class Create extends Setup {
+export class Create extends SfdxCommand implements HasRequirements {
     public static description = messages.getMessage('commandDescription');
 
     public static readonly flagsConfig: FlagsConfig = {
-        apilevel: flags.string({
-            char: 'a',
-            description: messages.getMessage('apiLevelFlagDescription'),
-            longDescription: messages.getMessage('apiLevelFlagDescription'),
-            required: false
-        }),
         devicename: flags.string({
             char: 'n',
             description: messages.getMessage('deviceNameFlagDescription'),
-            required: true
+            required: true,
+            validate: (deviceName) => {
+                if (deviceName.trim() === '') {
```

Comment:
I don't know if it will ever be the case where `deviceName` would have a value of `null` or `undefined` but I think it would be safer if we changed the logic here to something like this instead (and should do the same for other flags/commands):

```
if (deviceName && deviceName.trim().length > 0) {
    return true
} else {
    throw .....
}
```